core/templates/dev/head/components/entity-creation-services/collection-creation.service.ts

```diff
@@ -15,21 +15,18 @@
 /**
  * @fileoverview Modal and functionality for the create collection button.
  */
-
-require('domain/utilities/url-interpolation.service.ts');
+// eslint-disable-next-line max-len
+require('components/entity-creation-services/collection-creation-backend-api.service.ts');// lint-disable-line no-use-before-define
```

Comment:
This disabling is kind of messy. Also, why is there `// lint-disable-line no-use-before-define`?

Reply:
Sorry, that was a mistake. Removed it. Added an empty line to try to make it look clean. PTAL